slackbot-guysgals-reasoning.md

```diff
@@ -0,0 +1,34 @@
+# Why the Auto Response?
+
+The ZATech community has some people who feel excluded by certain words. In particular there was a point raised in #diversity about the word `guys`.
```

Comment:
I don't like this first sentence. I feel like it subtly puts blame on the people feeling excluded and makes it sound like the portion of people that feel excluded by the term, within the community or not, is smaller than it is.